guides/common/modules/proc_creating-an-insights-remediation-plan-for-hosts.adoc

```diff
@@ -1,19 +1,20 @@
 :_mod-docs-content-type: PROCEDURE
 
-[id="Creating_an_Insights_Remediation_Plan_for_Hosts_{context}"]
-= Creating an Insights remediation plan for hosts
+[id="creating-an-insights-remediation-plan-for-hosts"]
+= Creating an {Insights} remediation plan for hosts
 
-With {Project}, you can create a Red{nbsp}Hat Insights remediation plan and run the plan on {Project} hosts.
+With {Project}, you can create a {Insights} remediation plan and run the plan on {Project} hosts.
 
 .Procedure
 ifndef::satellite[]
-. In the {ProjectWebUI}, navigate to *Insights* > *Recommendations*.
+. In the {ProjectWebUI}, navigate to *{Insights}* > *Recommendations*.
+. On the {Insights} page, select the number of recommendations that you want to include in an {Insights} plan.
 endif::[]
 
 ifdef::satellite[]
-. In the {ProjectWebUI}, navigate to *Red{nbsp}Hat Lightspeed* > *Recommendations*.
+. In the {ProjectWebUI}, navigate to *{Insights}* > *Recommendations*.
+. On the {Insights} page, select the number of recommendations that you want to include in a {Insights} plan.
 endif::[]
-. On the Red{nbsp}Hat Insights page, select the number of recommendations that you want to include in an Insights plan.
```

Comment:
I think you can shorten this by dropping line 9-14, keeping line 15+16, and dropping line 17.